SEED-platform/seed
# 3178 feat/save r2 value

#### Any background context you want to provide?
see ticket

#### What's this PR do?
Saves the BETTEER R2 values to extra data and displays them on the analysiis highlight card.  Note:  displaying analysis results in detail on the analysis page is a future task.

#### How should this be manually tested?
Run a successful BETTER analysis and check out the saved values (via database or via highlight card)

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/3178

#### Screenshots (if appropriate)
<img width="1280" alt="Screen Shot 2022-05-16 at 2 10 26 PM" src="https://user-images.githubusercontent.com/564231/168674989-55213366-cebd-4081-bce1-04473910d436.png">
